index.js

```diff
@@ -0,0 +1,9 @@
+import ExportUsers from './src/ExportUsers'
```

Comment:
This module is untested because it's using code that's already covered in our test suite.